server/apps/main/templates/main/partials/footer.html

```diff
@@ -13,21 +13,21 @@
     <div class="col-lg-3 footer-text">
       <h5>Allied Organisations</h5>
```

Comment:
it feels weird to list Autistica, the turing & open humans as "allied organisations" given that 2-3 of them are not just allied but actually involved in running autspaces?

Reply:
I changed this to "Organisations which support AutSPACEs" - does that wording work or is it still a bit ambiguous?